test/test-fs.c

```diff
@@ -2257,3 +2257,75 @@ TEST_IMPL(fs_write_multiple_bufs) {
   MAKE_VALGRIND_HAPPY();
   return 0;
 }
+
+
+TEST_IMPL(fs_write_alotof_bufs) {
+  const size_t iovcount=54321;
+  uv_buf_t *iovs=malloc(sizeof(*iovs)*iovcount);
+  char *buffer;
+  size_t index;
+  int r;
+
+  /* Setup. */
+  unlink("test_file");
+
+  loop = uv_default_loop();
+
+  r = uv_fs_open(loop, &open_req1, "test_file", O_WRONLY | O_CREAT,
+      S_IWUSR | S_IRUSR, NULL);
```

Comment:
Some parts of the code may not be 100% consistent with the style we use now, but we try to make new code always match the reules @bnoordhuis outlined.